src/qt/privacydialog.cpp

```diff
@@ -439,17 +441,17 @@ void PrivacyDialog::sendzPIV()
     // Display errors during spend
     if (!fSuccess) {
         int nNeededSpends = receipt.GetNeededSpends(); // Number of spends we would need for this transaction
```

Comment:
``nNeededSpends`` is not used with these changes. Can be commented out or removed too.

Reply:
done.